subsys/fs/nvs/nvs.c

```diff
@@ -526,6 +526,19 @@ static int nvs_gc(struct nvs_fs *fs)
 	return 0;
 }
 
+static int nvs_mem_cmp_const(const u8_t *buf, u8_t const_val, size_t len)
```

Comment:
This routine is already available (nvs_ate_cmp_cnst), no need to add a new one.

Reply:
right, I didn't notice